src/main/kotlin/racingcar/domain/strategy/MovableStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar.domain.strategy
+
+interface MovableStrategy {
+    fun move(): Boolean
+}
```

Comment:
함수형 인터페이스 사용 👍 

Java에서는 `@FunctionalInterface` 애너테이션이 있듯이 kotlin에서는 `fun interface`가 있어요. 😃

```suggestion
fun interface MovableStrategy {
    fun move(): Boolean
}
```

Reply:
그렇군요!! 
추상 메소드가 단 1개가 있는 인터페이스의 경우 함수형 인터페이스를 활용하도록 하겠습니다!!

함수형 인터페이스를 사용하면, 따로 익명 클래스를 정의할 필요없이 람다식으로 대체가 가능하군요!